client/src/shared/layouts/Right Side Bar/RightSideBar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface rightSideBarProps {
+  children: any
+}
+
+const RightSideBar: React.FunctionComponent<rightSideBarProps> = ({
+  children
+}: rightSideBarProps) => {
+  return (
```

Comment:
please add the left container also, follow the layout [here](https://www.figma.com/file/6f92xbjEjJfQvwzbCTOWqF/TRAINING-PROGRAM-LMS?node-id=212-16&t=uIAvq0xwKPruFeG7-0)